packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -397,15 +412,15 @@ type RichText implements Widget {
 type FormDescription implements RepresentationDescription {
   id: ID!
   label: String!
-  completionProposals(widgetId: ID! currentText: String! cursorPosition: Int!): [CompletionProposal!]!
+  completionProposals(
+    widgetId: ID!
+    currentText: String!
+    cursorPosition: Int!
+  ): [CompletionProposal!]!
   helpText(widgetId: ID!): String
 }
 
-type CompletionProposal {
-  description: String!
-  textToInsert: String!
-  charsToReplace: Int!
-}
+type CompletionProposal { description: String! textToInsert: String! charsToReplace: Int! }
```

Comment:
Please let untouched code with the same formatting. Could you check everywhere else in this PR please?